build/vega-lite-schema.json

```diff
@@ -1011,7 +1011,10 @@
           "enum": [
             "filter"
           ],
-          "type": "string"
+          "type": [
+            "string",
+            "null"
```

Comment:
@domoritz  The schema generator actually still generate incorrect results here.

(The `enum` property should include `null` above -- otherwise, JSON schema validator would still throw warning.)

Reply:
Hmm, so you want it to be 

```
enum: "filter", null
type: string
```

?

Reply:
I think it should be 


```
enum: ["filter", null]
type: ["string", "null"]
```

Reply:
https://spacetelescope.github.io/understanding-json-schema/reference/null.html isn't clear about this but it seems that you are right: https://github.com/json-schema-org/json-schema-spec/issues/258. We do need to add the type and also add `null` to the enum. 

Fixed in https://github.com/vega/ts-json-schema-generator/issues/22

Reply:
Fixed. I added a commit with the new schema.